### PR TITLE
install clutter

### PR DESCRIPTION
**Pull request by itayd**
clutter is a tool to make cross referencing code comments possible. see https://github.com/cluttercode/clutter.
this is to be used for docs in https://github.com/envoyproxy/envoy/pull/15051.

### CODE REVIEW

**Comment by itayd** [comment]
huh, no repokitteh here

File: build_container/build_container_ubuntu.sh [hunk 1]
```diff
@@ -139,6 +139,15 @@ sudo cp shellcheck-v0.7.0/shellcheck /usr/local/bin
 rm -rf shellcheck-v0.7.0/ sc.checksum
 popd || exit 1
 
+# install clutter
+pushd /tmp || exit 1
+echo "9d5e80ae8da4b7bcbdb35356474a01079f0afa131dba04f9dbc95f27a26dbac6 /tmp/clutter" > clutter.checksum
+wget -qO- https://github.com/cluttercode/clutter/releases/download/0.2.1/clutter_0.2.1_Linux_x86_64.tar.gz | tar -xzf -
+sha256sum -c clutter.checksum
+sudo cp clutter /usr/local/bin
+rm -rf cutter sc.checksum
```
Comment (phlax) [inline build_container/build_container_ubuntu.sh:148]: `cutter` and `sc.checksum`

**Comment by phlax** [comment]
looks good apart from typos - once it lands you will need to bump the hashes in the main envoy repo

File: build_container/build_container_ubuntu.sh (hunk 1 above)
Comment (itayd) [reply build_container/build_container_ubuntu.sh]: fixed, but dco. going to push from clean.

Comment (phlax) [reply build_container/build_container_ubuntu.sh]: `cutter` also

Comment (itayd) [reply build_container/build_container_ubuntu.sh]: fixed

**Review by sunjayBhatia — COMMENTED** [review]
clearing my review requests, looks like @phlax has approved but this needs a `main` merge

**Comment by phlax** [comment]
@sunjayBhatia i think this needs to close - clutter is being added with bazel in envoy itself instead

**Comment by sunjayBhatia** [comment]
> @sunjayBhatia i think this needs to close - clutter is being added with bazel in envoy itself instead

sounds good, if you have powers to close this and think it should be feel free of course, if you don't i can close it as well

**Comment by phlax** [comment]
i dont have those powers in this repo

for ref the related envoy pr is here https://github.com/envoyproxy/envoy/pull/15225

i spoke with @itayd yesterday and i think he will take over that PR - i believe the blocker is using gazelle

**Pull request closed without merging**